integration-tests/lambda-view.spec.ts

```diff
@@ -47,6 +47,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
       await op.confirmation()
       const contract = await op.contract();
+      console.log("Originate FA1.2 contract and fetch data from view entrypoints contract address : "+contract.address)
```

Comment:
same as above

Reply:
ok./ i thought they might be useful for next time we want to re originate for a new protocol - I will remove them